### ISSUE TITLE
Maker - Environment Request canvas app

### ISSUE THREAD

**Issue by Minh-Truong** [microsoft/coe-starter-kit#5184]
### Discussed in https://github.com/microsoft/coe-starter-kit/discussions/5179

<div type='discussions-op-text'>

<sup>Originally posted by **Minh-Truong** April  3, 2023</sup>
I'm working for a client who is starting to have CoE toolkit in their tenant.  I have some questions regarding the Maker - Environment Request canvas app.
- This canvas app is using Office365 Get Groups connector to get security groups.  were there any reasonings behind using this connector?  We would like to know more about/
- Client considers to get AAD security groups instead of Office365 groups.  Any idea(s)/reasoning(s) should client should go or not go with this enhancement?  If we would have to enhance it, would you please let us know how big the effort would be?
- Does CoE toolkit team already have plan for both Office365 groups and AAD groups for this canvas app?</div>

**Comment by denise-msft** [comment]
@Minh-Truong 

- Using the Office365Groups connector to retrieve groups because that is an appropriate way to retrieve groups. We added the new graph call because the existing one did not return all groups. What is the issue behind this question?
- It was scoped to security groups because this was already an incremental improvement from what was there before. Are office 365 groups desired as the security group for limiting the environment security?
- There was no plan but it is not difficult to add - there is an odata filter on the ListGroups operation that chooses securityEnabled. Did you mean to add Microsoft 365 groups? (https://learn.microsoft.com/en-us/power-platform/admin/control-user-access#:~:text=Environments%20support%20associating%20the%20following%20group%20types%3A%20Security%20and%20Microsoft%20365.%20Associating%20other%20group%20types%20is%20not%20supported.)

**Comment by denise-msft** [comment]
@Minh-Truong are you only missing dynamic distribution groups, or are you missing other categories of groups?

The reason why we used office 365 users connector is because the Azure AD connector is often blocked by DLP policies - so the thinking was this would make managing DLP easier.

**Comment by Minh-Truong** [comment]
@denise-msft 
There were AAD groups client showed us in their Azure portal but they cannot/did not see using the Maker - Environment Request canvas app

**Comment by Minh-Truong** [comment]
@denise-msft , the distribution groups were missing

**Comment by mohamrizwa** [comment]
We noticed that this issue has been pending for quite some time. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved.
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue.
Thank you for your understanding and cooperation.
